Makefile

```diff
@@ -96,6 +96,6 @@ changelog:
 
 docker:
 	@$(call print, "Building docker image")
-	docker build -t boltz/boltz-lnd:$(version) .
+	docker buildx build --platform linux/amd64 --platform linux/arm64 --platform linux/arm/v7 -t boltz/boltz-lnd:$(version) -t boltz/boltz-lnd:latest .
```

Comment:
Which tags do you need for the docker image? Versions for the releases or `latest`?

Reply:
Ideally you would have both tags. That would allow for pinning applications to a specific tagged version as well as "automatically" updating to the latest version available when using the `latest` tag.

As an example, look at the many tags provide by [node on dockerhub](https://hub.docker.com/_/node). While this is certainly overkill, it shows the benefits of providing multiple tags for each image.